src/scripts/search-engines/qwant.ts

```diff
@@ -0,0 +1,146 @@
+import { SerpHandler } from '../types';
+import { handleSerp } from './helpers';
+
+export function getSerpHandler(): SerpHandler {
+  const serpHandler = handleSerp({
+    globalStyle: {
+      '[data-ub-blocked="visible"]': {
+        backgroundColor: 'var(--ub-block-color, rgba(255, 192, 192, 0.5)) !important',
+      },
+      '.theme-dark [data-ub-blocked="visible"], [data-theme="dark"] [data-ub-blocked="visible"]': {
+        backgroundColor: 'var(--ub-block-color, rgba(101, 8, 8, 0.5)) !important',
+      },
+      '.ub-button': {
+        color: 'var(--ub-link-color, rgb(93, 96, 94))',
+      },
+      '.theme-dark .ub-button, [data-theme="dark"] .ub-button': {
+        color: 'var(--ub-link-color, rgb(122, 140, 109))',
+      },
+      '.ub-button:hover': {
+        textDecoration: 'underline',
+      },
+      'div[class*=WebResult-module__permalink]': {
```

Comment:
`^=` may be better than `*=`. (Just a matter of taste?)

Reply:
It will actually not work (that was my first attempt) as there are many classes and this one for example is the latest.

Reply:
I see.